examples/python/drone_lidar/.gitignore

```diff
@@ -0,0 +1 @@
+/data
```

Comment:
All other examples calls the folder `dataset`

https://github.com/rerun-io/rerun/blob/73bbbabd317c6fe0c693f8dbb9f31bfc5dfc6bf2/.gitignore#L46